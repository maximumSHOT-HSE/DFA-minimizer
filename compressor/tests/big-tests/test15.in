92 828 9
1 7 0
1 12 1
1 23 2
1 47 3
1 23 4
1 10 5
1 22 6
1 46 7
1 25 8
2 44 0
2 48 1
2 22 2
2 25 3
2 50 4
2 20 5
2 16 6
2 26 7
2 27 8
3 6 0
3 49 1
3 39 2
3 12 3
3 32 4
3 21 5
3 36 6
3 15 7
3 54 8
4 23 0
4 16 1
4 52 2
4 25 3
4 12 4
4 12 5
4 41 6
4 52 7
4 20 8
5 13 0
5 25 1
5 4 2
5 31 3
5 4 4
5 46 5
5 42 6
5 33 7
5 46 8
6 34 0
6 54 1
6 10 2
6 43 3
6 16 4
6 14 5
6 15 6
6 20 7
6 23 8
7 53 0
7 20 1
7 40 2
7 12 3
7 1 4
7 31 5
7 8 6
7 25 7
7 52 8
8 10 0
8 47 1
8 51 2
8 21 3
8 26 4
8 51 5
8 18 6
8 39 7
8 29 8
9 16 0
9 42 1
9 1 2
9 36 3
9 4 4
9 33 5
9 21 6
9 49 7
9 20 8
10 20 0
10 29 1
10 24 2
10 26 3
10 7 4
10 51 5
10 18 6
10 9 7
10 21 8
11 12 0
11 3 1
11 33 2
11 10 3
11 52 4
11 9 5
11 47 6
11 28 7
11 20 8
12 47 0
12 2 1
12 43 2
12 1 3
12 2 4
12 39 5
12 10 6
12 10 7
12 37 8
13 38 0
13 31 1
13 33 2
13 18 3
13 18 4
13 32 5
13 16 6
13 26 7
13 15 8
14 4 0
14 46 1
14 39 2
14 10 3
14 17 4
14 23 5
14 29 6
14 1 7
14 24 8
15 25 0
15 39 1
15 8 2
15 21 3
15 41 4
15 20 5
15 7 6
15 30 7
15 4 8
16 16 0
16 40 1
16 1 2
16 31 3
16 35 4
16 54 5
16 28 6
16 26 7
16 40 8
17 8 0
17 44 1
17 36 2
17 22 3
17 8 4
17 48 5
17 50 6
17 18 7
17 21 8
18 23 0
18 10 1
18 52 2
18 11 3
18 50 4
18 12 5
18 50 6
18 29 7
18 52 8
19 12 0
19 29 1
19 16 2
19 8 3
19 35 4
19 34 5
19 25 6
19 48 7
19 39 8
20 18 0
20 18 1
20 19 2
20 33 3
20 29 4
20 36 5
20 50 6
20 41 7
20 33 8
21 43 0
21 19 1
21 46 2
21 3 3
21 24 4
21 35 5
21 12 6
21 1 7
21 7 8
22 44 0
22 5 1
22 36 2
22 35 3
22 40 4
22 42 5
22 37 6
22 3 7
22 37 8
23 40 0
23 50 1
23 52 2
23 27 3
23 47 4
23 3 5
23 4 6
23 25 7
23 51 8
24 15 0
24 23 1
24 19 2
24 27 3
24 13 4
24 3 5
24 36 6
24 9 7
24 42 8
25 37 0
25 9 1
25 38 2
25 16 3
25 34 4
25 3 5
25 18 6
25 20 7
25 3 8
26 28 0
26 10 1
26 41 2
26 37 3
26 1 4
26 22 5
26 49 6
26 36 7
26 28 8
27 45 0
27 49 1
27 41 2
27 48 3
27 38 4
27 3 5
27 7 6
27 26 7
27 39 8
28 22 0
28 48 1
28 51 2
28 37 3
28 32 4
28 39 5
28 39 6
28 4 7
28 38 8
29 49 0
29 37 1
29 30 2
29 44 3
29 42 4
29 54 5
29 40 6
29 35 7
29 23 8
30 16 0
30 26 1
30 24 2
30 10 3
30 6 4
30 9 5
30 19 6
30 24 7
30 48 8
31 38 0
31 22 1
31 32 2
31 37 3
31 15 4
31 53 5
31 32 6
31 28 7
31 37 8
32 36 0
32 37 1
32 8 2
32 52 3
32 50 4
32 18 5
32 30 6
32 12 7
32 11 8
33 29 0
33 36 1
33 33 2
33 25 3
33 23 4
33 40 5
33 6 6
33 51 7
33 30 8
34 8 0
34 6 1
34 24 2
34 17 3
34 24 4
34 40 5
34 24 6
34 14 7
34 26 8
35 4 0
35 15 1
35 25 2
35 2 3
35 27 4
35 28 5
35 14 6
35 41 7
35 39 8
36 4 0
36 31 1
36 46 2
36 26 3
36 29 4
36 11 5
36 20 6
36 18 7
36 15 8
37 7 0
37 25 1
37 6 2
37 30 3
37 38 4
37 30 5
37 22 6
37 15 7
37 24 8
38 47 0
38 10 1
38 37 2
38 21 3
38 36 4
38 21 5
38 1 6
38 7 7
38 52 8
39 1 0
39 3 1
39 36 2
39 30 3
39 14 4
39 48 5
39 53 6
39 7 7
39 11 8
40 34 0
40 40 1
40 9 2
40 11 3
40 20 4
40 42 5
40 2 6
40 52 7
40 50 8
41 31 0
41 46 1
41 44 2
41 9 3
41 32 4
41 39 5
41 39 6
41 50 7
41 35 8
42 46 0
42 42 1
42 12 2
42 28 3
42 32 4
42 28 5
42 53 6
42 3 7
42 44 8
43 47 0
43 2 1
43 29 2
43 11 3
43 53 4
43 26 5
43 12 6
43 28 7
43 42 8
44 23 0
44 53 1
44 39 2
44 23 3
44 15 4
44 8 5
44 41 6
44 37 7
44 10 8
45 36 0
45 18 1
45 16 2
45 54 3
45 21 4
45 6 5
45 17 6
45 15 7
45 9 8
46 24 0
46 41 1
46 20 2
46 46 3
46 49 4
46 47 5
46 29 6
46 43 7
46 47 8
47 54 0
47 29 1
47 10 2
47 11 3
47 41 4
47 13 5
47 52 6
47 1 7
47 20 8
48 36 0
48 29 1
48 3 2
48 50 3
48 50 4
48 30 5
48 19 6
48 25 7
48 13 8
49 1 0
49 30 1
49 45 2
49 35 3
49 34 4
49 41 5
49 10 6
49 35 7
49 1 8
50 47 0
50 50 1
50 16 2
50 47 3
50 49 4
50 14 5
50 19 6
50 17 7
50 41 8
51 7 0
51 32 1
51 43 2
51 25 3
51 7 4
51 38 5
51 32 6
51 49 7
51 20 8
52 40 0
52 11 1
52 35 2
52 54 3
52 14 4
52 19 5
52 11 6
52 43 7
52 2 8
53 50 0
53 42 1
53 49 2
53 47 3
53 47 4
53 1 5
53 28 6
53 39 7
53 4 8
54 48 0
54 19 1
54 11 2
54 43 3
54 15 4
54 21 5
54 4 6
54 47 7
54 51 8
55 53 0
55 20 1
55 40 2
55 12 3
55 1 4
55 31 5
55 8 6
55 25 7
55 52 8
56 47 0
56 10 1
56 37 2
56 21 3
56 36 4
56 21 5
56 1 6
56 7 7
56 52 8
57 23 0
57 16 1
57 52 2
57 25 3
57 12 4
57 12 5
57 41 6
57 52 7
57 20 8
58 12 0
58 3 1
58 33 2
58 10 3
58 52 4
58 9 5
58 47 6
58 28 7
58 20 8
59 36 0
59 29 1
59 3 2
59 50 3
59 50 4
59 30 5
59 19 6
59 25 7
59 13 8
60 7 0
60 25 1
60 6 2
60 30 3
60 38 4
60 30 5
60 22 6
60 15 7
60 24 8
61 54 0
61 29 1
61 10 2
61 11 3
61 41 4
61 13 5
61 52 6
61 1 7
61 20 8
62 1 0
62 30 1
62 45 2
62 35 3
62 34 4
62 41 5
62 10 6
62 35 7
62 1 8
63 47 0
63 2 1
63 43 2
63 1 3
63 2 4
63 39 5
63 10 6
63 10 7
63 37 8
64 7 0
64 32 1
64 43 2
64 25 3
64 7 4
64 38 5
64 32 6
64 49 7
64 20 8
65 15 0
65 23 1
65 19 2
65 27 3
65 13 4
65 3 5
65 36 6
65 9 7
65 42 8
66 54 0
66 29 1
66 10 2
66 11 3
66 41 4
66 13 5
66 52 6
66 1 7
66 20 8
67 12 0
67 3 1
67 33 2
67 10 3
67 52 4
67 9 5
67 47 6
67 28 7
67 20 8
68 47 0
68 10 1
68 37 2
68 21 3
68 36 4
68 21 5
68 1 6
68 7 7
68 52 8
69 10 0
69 47 1
69 51 2
69 21 3
69 26 4
69 51 5
69 18 6
69 39 7
69 29 8
70 16 0
70 42 1
70 1 2
70 36 3
70 4 4
70 33 5
70 21 6
70 49 7
70 20 8
71 53 0
71 20 1
71 40 2
71 12 3
71 1 4
71 31 5
71 8 6
71 25 7
71 52 8
72 22 0
72 48 1
72 51 2
72 37 3
72 32 4
72 39 5
72 39 6
72 4 7
72 38 8
73 36 0
73 29 1
73 3 2
73 50 3
73 50 4
73 30 5
73 19 6
73 25 7
73 13 8
74 38 0
74 31 1
74 33 2
74 18 3
74 18 4
74 32 5
74 16 6
74 26 7
74 15 8
75 7 0
75 32 1
75 43 2
75 25 3
75 7 4
75 38 5
75 32 6
75 49 7
75 20 8
76 7 0
76 32 1
76 43 2
76 25 3
76 7 4
76 38 5
76 32 6
76 49 7
76 20 8
77 34 0
77 54 1
77 10 2
77 43 3
77 16 4
77 14 5
77 15 6
77 20 7
77 23 8
78 16 0
78 42 1
78 1 2
78 36 3
78 4 4
78 33 5
78 21 6
78 49 7
78 20 8
79 37 0
79 9 1
79 38 2
79 16 3
79 34 4
79 3 5
79 18 6
79 20 7
79 3 8
80 16 0
80 42 1
80 1 2
80 36 3
80 4 4
80 33 5
80 21 6
80 49 7
80 20 8
81 4 0
81 31 1
81 46 2
81 26 3
81 29 4
81 11 5
81 20 6
81 18 7
81 15 8
82 48 0
82 19 1
82 11 2
82 43 3
82 15 4
82 21 5
82 4 6
82 47 7
82 51 8
83 53 0
83 20 1
83 40 2
83 12 3
83 1 4
83 31 5
83 8 6
83 25 7
83 52 8
84 31 0
84 46 1
84 44 2
84 9 3
84 32 4
84 39 5
84 39 6
84 50 7
84 35 8
85 36 0
85 29 1
85 3 2
85 50 3
85 50 4
85 30 5
85 19 6
85 25 7
85 13 8
86 47 0
86 50 1
86 16 2
86 47 3
86 49 4
86 14 5
86 19 6
86 17 7
86 41 8
87 12 0
87 29 1
87 16 2
87 8 3
87 35 4
87 34 5
87 25 6
87 48 7
87 39 8
88 47 0
88 10 1
88 37 2
88 21 3
88 36 4
88 21 5
88 1 6
88 7 7
88 52 8
89 34 0
89 54 1
89 10 2
89 43 3
89 16 4
89 14 5
89 15 6
89 20 7
89 23 8
90 7 0
90 12 1
90 23 2
90 47 3
90 23 4
90 10 5
90 22 6
90 46 7
90 25 8
91 16 0
91 42 1
91 1 2
91 36 3
91 4 4
91 33 5
91 21 6
91 49 7
91 20 8
92 15 0
92 23 1
92 19 2
92 27 3
92 13 4
92 3 5
92 36 6
92 9 7
92 42 8
12
41
58 53 22 78 10 79 55 39 23 80 15 44 28 81 6 61 27 70 46 35 75 56 2 71 54 24 48 19 20 16 73 74 9 85 86 31 60 47 84 21 13 
