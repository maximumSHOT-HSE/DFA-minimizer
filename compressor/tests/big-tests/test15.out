11 77 7
10 1 0
3 1 1
1 1 3
6 1 3
8 1 3
7 2 0
5 2 2
7 2 2
8 2 5
6 2 6
8 2 6
9 2 6
1 3 0
3 3 0
1 3 1
4 3 1
2 3 4
4 3 4
8 3 4
9 3 4
5 3 5
2 3 6
7 3 6
11 4 0
2 4 2
9 4 3
10 4 3
7 4 4
9 4 5
3 4 6
10 4 6
2 5 0
2 5 1
10 5 2
11 5 3
6 5 4
7 5 5
5 6 0
6 6 0
9 6 0
10 6 1
2 6 3
3 6 3
4 6 3
11 6 4
3 6 5
4 7 0
5 7 1
11 7 1
10 7 4
4 7 5
4 7 6
5 7 6
9 8 1
3 8 2
9 8 2
7 8 3
2 8 5
6 8 5
10 8 5
8 9 1
4 9 2
1 9 4
1 9 6
6 10 1
1 10 2
8 10 2
11 10 2
1 10 5
11 10 5
8 11 0
7 11 1
6 11 2
5 11 3
3 11 4
5 11 4
11 11 6
11
6
5 6 7 8 9 10 
